packaging/opensc.spec

```diff
@@ -211,9 +209,6 @@ rm %{buildroot}%{_mandir}/man1/opensc-notify.1*
 
 %config(noreplace) %{_sysconfdir}/opensc-%{_arch}.conf
 # Co-owned with p11-kit so it is not hard dependency
-%dir %{_datadir}/p11-kit
-%dir %{_datadir}/p11-kit/modules
-%{_datadir}/p11-kit/modules/opensc.module
```

Comment:
this lists the files in the RPM so this should stay in if the `make install` will correctly install the file

Reply:
I think we need to add a p11-kit-devel into the `BuildRequires` and then add the configure switch to make sure this module is correctly configured and installed?

Reply:
Yes, a build requirement would be the pkg-config files of p11-kit.

Reply:
Now it looks like it is installed correctly and the Packit ci fails with:
```
error: Installed (but unpackaged) file(s) found:
   /usr/share/p11-kit/modules/opensc.module
```
So please, get back these three lines. Then I think we are good to go.

Reply:
done